liblumen_eir_interpreter/src/native/maps.rs

```diff
@@ -0,0 +1,26 @@
+use liblumen_alloc::erts::term::Atom;
+use liblumen_alloc::erts::term::AsTerm;
+use lumen_runtime::otp::maps;
+
+use crate::module::NativeModule;
+
+pub fn make_maps() -> NativeModule {
+    let mut native = NativeModule::new(Atom::try_from_str("maps").unwrap());
+
+    native.add_simple(Atom::try_from_str("get").unwrap(), 3, |proc, args| {
+        assert!(args.len() == 3);
+        Ok(maps::get_3::native(proc, args[0], args[1], args[2]).unwrap())
+    });
+
+    native.add_simple(Atom::try_from_str("get").unwrap(), 2, |proc, args| {
+        assert!(args.len() == 2);
+        let default = Atom::try_from_str("nil").unwrap();
```

Comment:
I'm sure if I did some more hunting I'd find a better way to get the atom `nil`, especially because I have to dip into unsafe to turn it into a term, and that sounds unnecessary for something like `nil`.

Reply:
```rust
use liblumen_alloc::erts::term::atom_unchecked;

atom_unchecked("nil")
```

Reply:
Thanks!